guides.md

```diff
@@ -28,6 +29,165 @@ our values.
 - [Voicemail](#voicemail)
 - [References](#references)
 
+## Email
+
+### Creating new inboxes (Administrators)
+
+What you'll need:
+- Admin access to [our Mailcow email server](https://link.hypha.coop/email)
+- The following user info (from https://link.hypha.coop/employees):
+    - preferred email `handle@hypha.coop`
+    - preferred value for "Full Name"
+    - primary email (where they'd like `@hypha.coop` emails to be forwarded initially)
+    - Signal-friendly phone number
+
+_(Consider creating a [new GitHub issue in `hyphacoop/organizing`](https://github.com/hyphacoop/organizing/issues/new) to track email onboarding.)_
+
+1. Sign into Mailcow: https://link.hypha.coop/email
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of Mailcow login page" src="images/email-mailbox-1.png" />
```

Comment:
In other guides numbering starts with `-0`

Reply:
beeennnnnnn 🙄

Reply:
Not sure where I see the numbering at 0. but want to be clear that consistency (aka standardization) in printed material is not about being more like a computer to me. It's about reducing snags/points of friction that encumber the reader. 

We should treat this as technical documentation to help our future selves and co-op mates, see how write the docs thinks on style guides and consistency for instance: https://www.writethedocs.org/guide/writing/style-guides/